xds/src/main/java/io/grpc/xds/XdsClusterResource.java

```diff
@@ -430,9 +431,10 @@ static void validateCommonTlsContext(
     }
     String rootCaInstanceName = getRootCertInstanceName(commonTlsContext);
     if (rootCaInstanceName == null) {
-      if (!server) {
+      if (!server && !CommonTlsContextUtil.isUsingSystemRootCerts(commonTlsContext)) {
```

Comment:
Missing GRPC_EXPERIMENTAL_XDS_SYSTEM_ROOT_CERTS env var protection

Reply:
Done, but not yet fixed unit test.